python_modules/libraries/dagster-airlift/dagster_airlift/test/shared_fixtures.py

```diff
@@ -100,8 +101,9 @@ def stand_up_airflow(
         assert airflow_ready, "Airflow did not start within 30 seconds..."
         yield process
     finally:
-        # Kill process group, since process.kill and process.terminate do not work.
-        os.killpg(process.pid, signal.SIGKILL)
+        if psutil.Process(pid=process.pid).is_running():
+            # Kill process group, since process.kill and process.terminate do not work.
+            os.killpg(process.pid, signal.SIGKILL)
```

Comment:
Yea not related to this PR. The main change here is the psutil.Process().is_running, since this runs in a finally block it's possible that the process is dead